### DISCUSSION TITLE
Simulate mouse movement/press

### DISCUSSION THREAD

**Discussion by Kyemate** [TolikPylypchuk/SharpHook#30] [Q&A] [answered]
Hi I can't get the dpi awareness to work using duel monitors. I've been playing around with app.manifest and also tried running the Sample app but just mouse just jumps around randomly.

**Comment by TolikPylypchuk** [comment, marked as answer]
Hi! Thanks for asking! Unfortunately, libuiohook (and by extension, SharpHook) doesn't work well with multiple monitors, as described in [this issue](https://github.com/TolikPylypchuk/SharpHook/issues/25).

If you need this feature, then you should probably ask about it in the [original libuiohook repository](https://github.com/kwhat/libuiohook). Although I have to warn you that it's not currently under active development so I'm not sure when you'll get a response.

**Comment by TolikPylypchuk** [comment]
Hi @Kyemate! Multiple monitors are now supported on Windows so you should try it out!

**Reply by Kyemate** [reply]
Thanks for the update!
